spec/features/proposal_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 feature Event do
   let!(:conference) { create(:conference) }
+  let!(:registration_period) { create(:registration_period, conference: conference, start_date: Date.current) }
```

Comment:
:+1: you are right, the relevant test should also check if the current_path is correct.